src/parsing/Parse_rule.ml

```diff
@@ -974,6 +974,7 @@ and parse_extra (env : env) (key : key) (value : G.expr) : extra =
         match analyzer with
         | "entropy" -> R.CondEntropy
         | "redos" -> R.CondReDoS
+        | "entropy_v2" -> R.CondEntropyV2
```

Comment:
May require change in `semgrep_interfaces` depending on how we express in schema. (I think *probably* not, given my assumption is that we just say this is a string, but it could be an enum there).

Reply:
Unrelated: may be better to keep this in sorted order.

Reply:
@mjambon to confirm but I am 99% sure this is just a string based on poring over the code

Reply:
I think this is ok, the only reference I could find to entropy in https://github.com/returntocorp/semgrep-interfaces/blob/main/rule_schema_v1.yaml was a comment.

Reply:
@kopecs given that this builds + runs - are you okay w/leaving this for now? @akuhlens can file some cleanup tickets for "niceness"

Reply:
Yes, this lgtm. I just wasn't sure if https://github.com/returntocorp/semgrep-interfaces/blob/28ad3b3c4afe67083d759e8d8feefed56b440a8d/rule_schema_v1.yaml#L407 was in fact string (it is) or if it was an enum.